src/site/xdoc/server/config-webadmin.xml

```diff
@@ -30,6 +30,31 @@
     <p>Consult <a href="https://github.com/apache/james-project/server/container/guice/memory-guice/sample-configuration/webadmin.properties">webadmin.properties</a> in GIT to get some examples and hints.</p>
 
     <p>Use this configuration to define the WebAdmin's responding host and port.</p>
+
+    <dl>
+        <dt><strong>enabled</strong></dt>
+        <dd>Define if WebAdmin is launched (default: false)</dd>
+        <dt><strong>port</strong></dt>
+        <dd>Define WebAdmin's port (default: 8080)</dd>
+        <dt><strong>host</strong></dt>
+        <dd>Define WebAdmin's host (default: localhost)</dd>
+        <dt><strong>cors.enable</strong></dt>
+        <dd>Allow the Cross-origin resource sharing (default: false)</dd>
+        <dt><strong>cors.origin</strong></dt>
+        <dd>Specify ths CORS origin(default: null)</dd>
+        <dt><strong>jwt.enable</strong></dt>
+        <dd>Allow JSON Web Token(default: false)</dd>
```

Comment:
idem

Reply:
and I guess there is one parameter missing then (the JWT public key), no?

Reply:
I can't find any another `jwt` option in the `WebAdmin` module, maybe you guessed `jwt.publickeypem.url` but it's in `Jmap` module

Reply:
oh interesting, it seems we have an issue here. The JWT public key is gotten though injection and in fact we use the one configured via JMAP. This is bad because it seems we cannot configure JWT webadmin without JMAP.